src/flash.rs

```diff
@@ -41,7 +41,12 @@ impl FLASH {
         // following should determine whether this is a Category 5 device.
         // Please make sure to adapt this when porting this module to other
         // targets.
-        let is_category_5 = cfg!(any(feature = "stm32l072", feature = "stm32l082"));
+        let is_category_5 = cfg!(any(
+            feature = "stm32l072",
+            feature = "stm32l082",
+            feature = "stm32l073",
+            feature = "stm32l083"
+        ));
```

Comment:
Maybe you could also update the comment above?

In the STM32L0x3 reference manual (RM0367), that information is also in section 1.4.